src/description.cpp

```diff
@@ -132,11 +132,38 @@ Description::Description(const string &sdp, Type type, Role role)
 			} else if (key == "candidate") {
 				addCandidate(Candidate(attr, currentMedia ? currentMedia->mid : mData.mid));
 			} else if (key == "end-of-candidates") {
-				mEnded = true;
+                mEnded = true;
```

Comment:
The project uses tabs to align while you used spaces. Not a big deal, but could you please retab the files so it displays nicely?